layouts/_default/baseof.html

```diff
@@ -23,6 +23,8 @@
     {{ if .Site.Params.cookie_consent_info_url }}
     {{ partial "cookie-consent.html" . }}
     {{ end }}
+
+		{{ partial "head-custom.html" . }}
```

Comment:
Woops. Looks like vim decided to use tabs to indent and I forgot to check. Do you mind if I add a modeline comment to the bottom of the file while I'm at it?